pages/all.js

```diff
@@ -10,7 +10,7 @@ import Layout from "@components/Layout";
 import Loader from "@components/Loader";
 import Switch from "@components/Switch";
 
-export default function New() {
+export default function All() {
   const router = useRouter(); // Setup router
 
   // Global state
```

Comment:
filterNewProposals  in all.js can be removed

Reply:
fixed, thanks